code/game/gamemodes/umbrage/umbrage_minor_abilities.dm

```diff
@@ -45,23 +45,25 @@
 	var/victims = list() //A list of people we've used the bead on recently; we can't drain them again so soon
 
 /datum/action/innate/umbrage/devour_will/IsAvailable()
-	if(!usr || !usr.get_empty_held_indexes())
+	var/mob/living/user = usr
```

Comment:
No this isn't what I meant at all, you've changed nothing!

USR IS NEVER VALID IN PROCS.
NEVER.

actions have an ```owner``` variable, if that's for some reason not being set for your actions then you're using actions wrong.